comp/core/workloadmeta/filter.go

```diff
@@ -5,27 +5,38 @@
 
 package workloadmeta
 
+// EntityFilterFunc provides a filter on the entity object level
+//
+// Given an entity instance, it returns true if the object should be
+// included in the output, and false if it should be filtered out.
+type EntityFilterFunc func(*Entity) bool
```

Comment:
Should we also replace the existing `ContainerFilterFunc`, `ProcessFilterFunc` and `GetRunningContainers` with this?